test/integration_test.go

```diff
@@ -839,6 +840,19 @@ func TestPriorityOrderOfArgument(t *testing.T) {
 	assert.Contains(t, out.String(), fmt.Sprintf("test = %s", injectedValue))
 }
 
+//func TestAutoRetryRerun(t *testing.T) {
```

Comment:
How come this is commented out?

Reply:
I commented it out just before the commit. Will uncomment.